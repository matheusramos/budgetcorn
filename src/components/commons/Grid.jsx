/* @flow */
import React from 'react';

export default (props: { children: React.Node }) => {
  const { children } = props;
  return (
    <div className='columns is-centered'>
      <div className='column is-full-mobile is-full-tablet is-half-desktop is-one-third-widescreen is-one-quarter-fullhd'>
        {children}
      </div>
    </div>
  );
};
