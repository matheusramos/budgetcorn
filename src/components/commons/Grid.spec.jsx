/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';

import Grid from './Grid';

it('renders correctly', () => {
  const tree = renderer.create(<Grid>Test</Grid>).toJSON();
  expect(tree).toMatchSnapshot();
});
