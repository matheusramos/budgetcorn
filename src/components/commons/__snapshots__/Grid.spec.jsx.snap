// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="columns is-centered"
>
  <div
    className="column is-full-mobile is-full-tablet is-half-desktop is-one-third-widescreen is-one-quarter-fullhd"
  >
    Test
  </div>
</div>
`;
